controller_manager/include/controller_manager/controller_manager.hpp

```diff
@@ -378,6 +379,9 @@ class ControllerManager : public rclcpp::Node
     const std::vector<ControllerSpec> & controllers, int strictness,
     const ControllersListIterator controller_it);
 
+  void CreateActiveDiagnostic(diagnostic_updater::DiagnosticStatusWrapper & stat);
```

Comment:
We are using rather `snake_case` in the names of the methods.

Is this maybe a better (clearer) name?
```suggestion
  void controller_activity_diagnostic_callback(diagnostic_updater::DiagnosticStatusWrapper & stat);
```